labelbox/data/serialization/coco/instance_dataset.py

```diff
@@ -123,6 +124,7 @@ def bbox_to_common(class_annotations: COCOObjectAnnotation,
     return annotations
 
 
+@retry()
```

Comment:
can you add stop=stop_after_attempt(20)

Reply:
and wait=wait_fixed(2)

Reply:
But also- I don't think this is gonna work. The retry needs to decorate the function that makes the http call, not this big one. So I assume get_image?